bin/chain-bootstrapper/src/chain_spec/builder.rs

```diff
@@ -116,7 +116,7 @@ fn generate_genesis_config(
         "staking": {
             "forceEra": Forcing::NotForcing,
             "validatorCount":  account_session_keys.len() as u32,
-            "minimumValidatorCount": 4,
+            "minimumValidatorCount": 1,
```

Comment:
I hate this and I don't trust anything to work with <4 validators, but you can just resolve this comment as I think I have lost this fight already. :P